### PR TITLE
✨ Compact SingleSelect & MultiSelect

### LINKED ISSUE #1035: Compact Select
- [ ] Adjust height to fit compact

- [ ] Empty label needed to fit to compact, can column name act as label for screen readers?

- [ ] How to fix selects in narrow columns

### PR DESCRIPTION
**Pull request by mimarz**
resolves #1035

### CODE REVIEW

File: libraries/core-react/src/components/Select/Select.tokens.ts [hunk 1]
```diff
@@ -62,4 +83,12 @@ export const multiSelect: ComponentToken = R.mergeDeepRight(select, {
     left: spacingMediumSmall,
     right: spacingLarge,
   },
+  modes: {
+    compact: {
+      spacings: {
+        top: '2px',
```
Comment (pomfrida) [inline libraries/core-react/src/components/Select/Select.tokens.ts:89]: Should we probably add `xx_small` (2px) here?

File: libraries/core-react/src/components/Select/Select.tokens.ts [hunk 2]
```diff
@@ -53,6 +54,26 @@ export const select: ComponentToken = {
...
+    compact: {
+      spacings: {
+        left: spacingSmall,
+        right: spacingSmall,
+        top: spacingSmall,
+        bottom: spacingSmall,
+      },
+      entities: {
+        button: {
+          height: '24px',
```
Comment (pomfrida) [inline libraries/core-react/src/components/Select/Select.tokens.ts:67]: No shape token for compact button? 🤔

**Review by pomfrida — APPROVED** [review]
Just some minor questions, LGTM otherwise 👍

File: libraries/core-react/src/components/Select/Select.tokens.ts (hunk 1 above)
Comment (mimarz) [reply libraries/core-react/src/components/Select/Select.tokens.ts:89]: Good point! Fixed 👌

File: libraries/core-react/src/components/Select/Select.tokens.ts (hunk 2 above)
Comment (mimarz) [reply libraries/core-react/src/components/Select/Select.tokens.ts:67]: Its actually not used. So removed :)

**Pull request merged**